File: snippets/cpp/VS_Snippets_CLR/DateTime.Add/CPP/class1.cpp
using namespace System;

int main()
{
   // <Snippet1>
   // Calculate what day of the week is 36 days from this instant.
   System::DateTime today = System::DateTime::Now;
   System::TimeSpan duration( 36, 0, 0, 0 );
   System::DateTime answer = today.Add( duration );
   System::Console::WriteLine( " {0:dddd}", answer );
   // </Snippet1>
}

File: snippets/csharp/System/DateTime/AddDays/class1.cs
// <Snippet1>
using System;

class Class1
{
    static void Main()
    {
        DateTime today = DateTime.Now;
        DateTime answer = today.AddDays(36);
        Console.WriteLine($"Today: {today:dddd}");
        Console.WriteLine($"36 days from today: {answer:dddd}");
    }
}
// The example displays output like the following:
//       Today: Wednesday
//       36 days from today: Thursday
// </Snippet1>
